db/seeds.rb

```diff
@@ -23,6 +27,12 @@
 racket = dog_shop.items.create!(name: "Tennis Racket", description: "Great Tennis Racket!", price: 200, image: "http://lvencaretoys.com/image/cache/dog/tu-toy-dog-pull-9010_2-800x800.jpg", inventory: 10)
 bell = dog_shop.items.create!(name: "Bell", description: "They will hear you comming", price: 30, image: "https://www.ri.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 9)
 helmet = dog_shop.items.create!(name: "Helmet", description: "Keep that head of yours safe", price: 150, image: "https://www.ri.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 32)
+pull_toy = bike_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
+dog_bone = bike_shop.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
+tennis_ball = a.items.create!(name: "Tennis Ball", description: "Great ball!", price: 5, image: "http://lovencaretoys.com/image/cache/dog/tu-toy-dog-pull-9010_2-800x800.jpg", inventory: 40)
+racket = b.items.create!(name: "Tennis Racket", description: "Great Tennis Racket!", price: 200, image: "http://lvencaretoys.com/image/cache/dog/tu-toy-dog-pull-9010_2-800x800.jpg", inventory: 10)
+bell = .items.create!(name: "Bell", description: "They will hear you comming", price: 30, image: "https://www.ri.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 9)
+helmet = c.items.create!(name: "Helmet", description: "Keep that head of yours safe", price: 150, image: "https://www.ri.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 32)
```

Comment:
Thanks for adding all of this to the seeds!

Reply:
Yeah thats really going to help!